docs/contexts/traceability-v1.jsonld

```diff
@@ -14,16 +14,16 @@
       "@id": "https://w3id.org/traceability/AgInspectionReport",
       "@context": {
         "facility": {
-          "@id": "https://schema.org/Place"
+          "@id": "https://w3id.org/traceability/#facility"
         },
         "inspector": {
-          "@id": "https://w3id.org/traceability/schemas/inspector"
+          "@id": "https://w3id.org/traceability/#inspector"
         },
         "shipment": {
           "@id": "https://schema.org/ParcelDelivery"
         },
-        "applicantId": {
-          "@id": "https://schema.org/identifier"
+        "applicant": {
+          "@id": "https://w3id.org/traceability/#dfn-entities"
```

Comment:
its possible to use anchor tags to remove the "dfn" part of this, for example:
https://github.com/w3c/did-spec-registries/blob/master/contexts/did-v1.jsonld#L21
https://www.w3.org/ns/activitystreams/#alsoKnownAs
https://www.w3.org/TR/did-core/#dfn-alsoknownas

https://www.w3.org/TR/did-core/#alsoKnownAs

Reply:
this is non blocking, but we might want to address in README guidance